src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -12,18 +12,20 @@
 @NoArgsConstructor
 @Entity(name = "CARTS")
 public class Cart {
+    public Cart(User user) {
```

Comment:
moim zdaniem ten konstruktor jest niepotrzebny, po założeniu koszyka chyba lepiej to ustawić setterem (ale przy zachowaniu odpowiedniej kolejności z tym konstruktorem chyba też może być, więc upierał się nie będę). Poza tym wprowadza konflikt kiedy używasz CascadeTyle.ALL i np. kiedy w testach zapisujesz usera z automatu zapisujesz też cart, a bez CascadeTyle.ALL musisz najpierw stworzyć i zapisać oddzielnie usera i cart, żeby potem na tym operować. A jak już ustaliliśmy CascadeTypa.ALL usuwamy :)

Reply:
Ja akurat używam tego konstruktora